docs/source/notebooks/08_Getting_and_bias_correcting_CMIP6_data.ipynb

```diff
@@ -0,0 +1,1788 @@
+{
```

Comment:
<p>"from the climate model simulations." instead of "climate model data" to avoid repeating data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>thanks! corrected.</p>